backend-server/collections/suspects.js

```diff
@@ -21,8 +21,8 @@ exports.suspects = [
     startingLocation: locations.find(location => location.name === 'Conservatory-Library-Hallway'),
     color: 'blue'
   }, {
-    name: 'Mrs. White',
+    name: 'Dr. Orchid',
```

Comment:
I didn't know they made this change. I wonder why? To be more inclusive?

Reply:
I think it was a refresh to publish a new version, like how Monopoly is updating their community chest cards. But she is the one that is in the version my parents have so I updated to match the cards.